libs/utils/decorate.js

```diff
@@ -60,11 +60,11 @@ export function decorateBlockText(el, config = ['m', 's', 'm'], type = null) {
         decorateIconArea(el);
       }
     }
-    const emptyPs = el.querySelectorAll(':scope p:not([class])');
+    const emptyPs = el.querySelectorAll('p:not([class]), ul:not([class]), ol:not([class])');
```

Comment:
Would it make sense to update the variable name, since it no longer exclusively refers to paragraph elements?